### ISSUE TITLE
Web client in System Scheme has got template variable {editTimeLimitButton} instead of number of sec

### ISSUE THREAD

**Issue by sCmEcOLJUcL** [mattermost/mattermost#27611]
#### Summary
Web client in System Scheme has got template variable {editTimeLimitButton} instead of number of sec

#### Steps to reproduce
/admin_console/user_management/permissions/system_scheme

Config json:
```
{
    "ServiceSettings": {
...
        "PostEditTimeLimit": 300,
...
}
}
```

#### Expected behavior
Mattermost ver. 9.9.0/9.9.1, see screenshot
![ksnip_20240710-121959](https://github.com/mattermost/mattermost/assets/29278617/cd787bce-a29f-4833-b8b4-216a78e03582)

**Comment by amyblais** [comment]
Ticket: https://mattermost.atlassian.net/browse/MM-58529.

**Comment by felibb** [comment]
Hey, I cannot comment in Mattermost JIRA, but this is quite relevant. What about porting that fix to 9.11 ESR?
